index/upsidedown/upsidedown.go

```diff
@@ -134,18 +134,23 @@ func (udc *UpsideDownCouch) loadSchema(kvreader store.KVReader) (err error) {
 	return
 }
 
+type RowBuffer struct {
```

Comment:
No @abhinavdangeti the `rowBuffer` struct does not need to be exported since it is only used within the same package. I have renamed it to `rowBuffer` because there is no need to export it.